frontend/src/utils/hooks/useScrollToAnchor.tsx

```diff
@@ -0,0 +1,19 @@
+import { useCallback } from 'react'
+
+export function useScrollToAnchor() {
+		const scrollToAnchor = useCallback((id: string, offset: number = -70) => {
+			const element = document.getElementById(id)
+
+			if (element) {
+				const yOffset = offset
+				const y =
+					element.getBoundingClientRect().top + window.pageYOffset + yOffset
+
+				window.scrollTo({ top: y, behavior: 'smooth' })
+			} else {
+				console.warn(`Element with id ${id} not found.`)
+			}
```

Comment:
this is ok while it's in dev, but before we release this feature we should remove any user facing console logs